test/unit-test/VssDatabaseTests.cpp

```diff
@@ -148,22 +148,22 @@ BOOST_AUTO_TEST_CASE(Given_ValidVssFilenameAndChannelAuthorized_When_GetBranch_S
 
   // setup
   db->initJsonTree(validFilename);
-  std::string signalPath{"Vehicle.Acceleration"};
+  VSSPath signalPath = VSSPath::fromVSSGen1("Vehicle.Acceleration");
 
   channel.setConnID(11);
   channel.setAuthorized(true);
 
   // expectations
 
-  MOCK_EXPECT(accCheckMock->checkReadDeprecated)
+  MOCK_EXPECT(accCheckMock->checkReadAccess)
     .returns(true);
 
-  std::string expectedJsonString{R"({"value":{"Vehicle.Acceleration.Lateral":"---","Vehicle.Acceleration.Longitudinal":"---","Vehicle.Acceleration.Vertical":"---"}})"};
+  std::string expectedJsonString{R"({"value":[{"Vehicle.Acceleration.Vertical":"---"},{"Vehicle.Acceleration.Longitudinal":"---"},{"Vehicle.Acceleration.Lateral":"---"}]})"};
```

Comment:
pls document such api changes

Reply:
As the original behaviour was not documented, and the V1 spec does not specify this, I suggest not documenting the "change" but rather provide an exhaustive documentation of the expected valid queries and responses, as we have done for "get" (and yes, that is still missing)